setup.py

```diff
@@ -98,7 +98,7 @@
         "finance", "eia 923", "eia 860", "ferc", "form 1", "epa ampd",
         "epa cems", "coal", "natural gas", "eia 861", "ferc 714"],
     python_requires=">=3.8,<3.10",
-    # setup_requires=["setuptools_scm"],
+    setup_requires=["setuptools_scm"],
```

Comment:
Why was this uncommented? It looks like this [usage has been deprecated](https://github.com/pypa/setuptools_scm#setuppy-usage-deprecated).

Reply:
I just tested it again with the extra line they say to add -- the `[tools.setuptools_scm]` section header. And it still doesn't work, at least within Tox. So I think there's just lag in all the various packaging tools picking up on the `pyproject.toml` settings (which it seems like has been in place for years at this point), so I think we have the same thing specified in both places for maximal compatibility.